docs/reference/glossary.md

```diff
@@ -45,6 +45,15 @@ Show page templates (`show.html`), do have access to their parent index page's `
 
 There is a single doc with the slug, `global`, which is always loaded and available to page templates as `data.global`. This is useful for shared, site-wide headers and footers that are editable, etc. It is managed by the `@apostrophecms/global` module.
 
+## Slug
+
+The slug of an Apostrophe doc is a string that uniquely identifies that doc within the database. Apostrophe will enforce that no two docs may have the same slug. Unlike the `_id` property of the database document, slugs _can_ change, however. Apostrophe user interface and REST API will suggest slugs automatically based on the document's title on creation.
```

Comment:
"no two docs may" remove "may"

"slugs _can_ change, however", remove "however". Also consider reversing. "Slugs can change, unlike the `id`", for example. (https://www.learngrammar.net/english-grammar/comparatives)

Reply:
**The** Apostrophe user interface